packages/datastore/datastore-codegen/tests/__snapshots__/Cli.test.ts.snap

```diff
@@ -0,0 +1,107 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`generate 1`] = `
+"export interface Note {
+    id: string;
+    title: string;
+    description: string;
+    comments: string[]
```

Comment:
This probably makes no sense. We should detect relationships and decompose them. 
That is why Delta type is much better as it gives us that already.

Reply:
Bumping this.